pinot-common/src/main/java/org/apache/pinot/common/metrics/AbstractMetrics.java

```diff
@@ -448,19 +458,6 @@ public void addValueToGlobalGauge(final G gauge, final long unitCount) {
     }
   }
 
-  /**
-   * Gets the value of a table gauge.
-   *
-   * @param tableName The table name
-   * @param gauge The gauge to use
-   */
-  public long getValueOfTableGauge(final String tableName, final G gauge) {
```

Comment:
I'm assuming that this has no usage anywhere else?

Reply:
Right, there is no usage.